src/components/modals/ManagePlanDerivationGroupsModal.svelte

```diff
@@ -0,0 +1,362 @@
+      valueFormatter: params => {
+        return params?.value.length;
+      },
+    },
+    {
+      cellRenderer: (params: DerivationGroupCellRendererParams) => {
+        var input = document.createElement('input');
+        input.type = 'checkbox';
+        input.checked = $selectedPlanDerivationGroupNames.includes(params?.data?.name ?? '');
+        input.addEventListener('click', event => {
+          if (event?.target && params.data) {
+            if ((event.target as any).checked) {
+              // insert
+              effects.insertDerivationGroupForPlan(params.data.name, $plan, user);
+              if ($derivationGroupPlanLinkError !== null) {
+                showFailureToast('Failed to link derivation group & plan.');
+              } else {
+                // Insert all the external event types from the derivation group to the timeline filter
+                const derivationGroup = $derivationGroups.find(
+                  derivationGroup => derivationGroup.name === params?.data?.name,
+                );
+                if (derivationGroup !== undefined) {
+                  externalEventLayers?.forEach(externalEventLayer => {
+                    if (externalEventLayer.filter.externalEvent !== undefined) {
+                      externalEventLayer.filter.externalEvent.event_types =
+                        externalEventLayer.filter.externalEvent.event_types.concat(derivationGroup.event_types);
+                      externalEventLayer.filter.externalEvent.event_types =
+                        externalEventLayer.filter.externalEvent.event_types.filter(
+                          (val, ind, arr) => arr.indexOf(val) === ind,
+                        ); // uniqueness
+                    }
+                  });
+                }
+              }
+            } else {
+              // delete
+              effects.deleteDerivationGroupForPlan(params.data.name, $plan, user);
+              if ($derivationGroupPlanLinkError !== null) {
+                showFailureToast('Failed to unlink derivation group & plan.');
+              }
+            }
+          }
+        });
+        return input;
+      },
+
+      // useful to set to true if two concurrent users at play, because if another user makes a change you see 
+      //    the cell highlighted, indicating that someone changed the cell. unfortunately, this flash occurs 
+      //    any time this cell is updated, meaning there's a flash when the store loads when this modal is opened, 
+      //    as well as a flash whenever a user changes it just in their own browser. As a result, it is more 
+      //    confusing than simple. There doesn't appear to be another good way to do it (as onCellValueChanged 
+      //    never fires, or else we could've done some diffing method with a local copy to determine if a change
+      //    is or is not foreign; it also doesn't look to be possible in cellRenderer).
+      enableCellChangeFlash: false, 
```

Comment:
`enableCellChangeFlash` should already be `false` by default. I'd say if that if it is necessary to draw attention to it in terms of design, then I'd suggest creating a design ticket so that we can take a look at whether or not we should flash the cell. Otherwise, I'd say it's safe to delete this and the comment above it.

Reply:
How should we create this design ticket, as this is an active PR? To leave the intention is, is it not better to just leave the comment in?

Reply:
The question I'd ask is who is the intended audience for this comment, and what should they do when they read it?

This code is setting a configuration to its default value - which doesn't seem to require an explanation (it's the default value, after all!). If you were changing the default value for some non-obvious reason, that would indeed require an explanation. Alternatively, a comment of this sort could go where the default is defined, and explain why it's the default.

The intent of the comment seems to be to spark discussion about design - and its scope goes beyond this PR (this isn't the only multi-user editable checkbox in Aerie, after all). Left where it is, it would be left for a developer down the line to stumble into, rather than captured where @lklyne and others can see it.

You can capture your ideas about users seeing each others' changes in a ticket with the design label - this link should start one for you: https://github.com/NASA-AMMOS/aerie-ui/issues/new?assignees=&labels=design

Reply:
Okay! Would it make sense in that design ticket to reference this PR in that ticket?

Reply:
Submitted an issue [here](https://github.com/NASA-AMMOS/aerie-ui/issues/1453).